packages/neos-ui-editors/src/Library/LinkInput.js

```diff
@@ -80,11 +81,16 @@ export default class LinkInput extends PureComponent {
     };
 
     getDataLoaderOptions() {
+        const contextNode = this.props.options.startingPoint ?
```

Comment:
```suggestion
        const contextNode = $get('options.startingPoint', this.props) ?
```

Reply:
This should be safer, in case there are no options

Reply:
done